librz/core/cmd/cmd_help.c

```diff
@@ -430,6 +430,104 @@ RZ_IPI void rz_core_clippy_print(RzCore *core, const char *msg) {
 	}
 }
 
+RZ_API void rz_core_cmd_help_calc_expr(RZ_NONNULL RzCore *core, RZ_NONNULL const char *input) {
```

Comment:
1. No need for it to be and API, IPI will suffice.
2. I recommend to use rzshell `intc argc, char **argv`-style arguments, also with `RzCmdStateOutput` for using JSON and other kinds of output

Reply:
Just make it `IPI` for this PR, it can be converted later then.

Reply:
set this as `RZ_IPI` and move the header definition under `core_private.h`